core/src/main/java/io/grpc/EquivalentAddressGroup.java

```diff
@@ -60,6 +61,33 @@ public EquivalentAddressGroup(SocketAddress addr) {
   }
 
   /**
+   * Converts list of ResolvedServerInfoGroup objects into one EquivalentAddressGroup object.
+   */
+  public static EquivalentAddressGroup fromResolvedServerInfoGroups(
```

Comment:
This logic should only be used by DummyLoadBalancer and thus should stay there. (I can't imagine another loadbalancer would use it).